tests/utils/verification/volumeproperties.go

```diff
@@ -96,6 +96,12 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// CheckVolumeAccessibility returns true if the given volume is accessible
+// from the specified VM; false otherwise.
+func CheckVolumeAccessibility(hostName string, volumeName string) bool {
+	return GetVolumeStatusHost(volumeName, hostName) != ""
```

Comment:
By accessible, it means the docker volume inspect (to get status of volume) from the given docker host was successful. Is this what you meant?